bullet/src/BulletDynamics/Dynamics/btRigidBody.h

```diff
@@ -278,6 +278,11 @@ class btRigidBody  : public btCollisionObject
 		m_totalForce += force*m_linearFactor;
 	}
 
+	void			applyCentralLocalForce(const btVector3& force)
```

Comment:
indentation looks wrong here

also, please add a comment `// XXX AMMO local addition to bullet`, so that when we update bullet, we will see it in the merge diffs and know what to do.